packages/vscode/src/util.ts

```diff
@@ -12,11 +12,36 @@ import {
   denyListDarkColorThemes,
   denyListLightColorThemes,
 } from './denyListColorThemes'
+import { homedir } from 'os'
+import { readdirSync } from 'fs'
+import path from 'path'
 
 export function isDebugOrTestSession(): boolean {
   return env.sessionId === 'someValue.sessionId'
 }
 
+export function checkForOtherPrismaExtension(extensionId: string) {
+  let watchedFolderPrefix = ''
+  if (extensionId === 'prisma.prisma-insider') {
+    watchedFolderPrefix = 'prisma.prisma-'
+  } else {
+    watchedFolderPrefix = 'prisma.prisma-insider-'
+  }
+
+  const files = readdirSync(
+    path.join(homedir(), '.vscode/extensions'),
```

Comment:
- Is this path like this for all OSes? 
- Can this path be changed/configured by the user?

Reply:
Yes, this path is the same for all OSes (more info: https://code.visualstudio.com/docs/editor/extension-gallery#_where-are-extensions-installed)

From that link: `You can change the location by launching VS Code with the --extensions-dir <dir> command-line option.`. We will not be able to reach those, but I think the majority of people will not use this CLI option.